packages/preview/arxivify/0.1.0/template/main.typ

```diff
@@ -0,0 +1,137 @@
+#import "@preview/arxivify:0.1.0": arxivify, arxivify_appendices
+
+#show: arxivify.with(
+  title: "ArXiv Typst Template",
+  authors: (
+    (name: "Author 1", email: "user@domain.com", affiliation: "Company", orcid: "0000-0000-0000-0000"),
+    (name: "Author 2", email: "user@domain.com", affiliation: "Company"),
+  ),
+  // Insert your abstract after the colon, wrapped in brackets.
+  // Example: `abstract: [This is my abstract...]`
+  abstract: lorem(55),
+  keywords: ("First keyword", "Second keyword", "etc."),
+  date: "May 16, 2023",
+)
+#set cite(style: "chicago-author-date")
+#show link: underline
+
+= Introduction
+#lorem(60)
+
+= Heading: first level
+#lorem(20)
+
+== Heading: second level
+#lorem(20)
+
+=== Heading: third level
+
+==== Paragraph
+#lorem(20)
+
+#lorem(20)
+
+= Math
+
+*Inline:* Let $a$, $b$, and $c$ be the side
+lengths of right-angled triangle. Then, we know that: $a^2 + b^2 = c^2$
+
+*Block without numbering:*
+
+#math.equation(block: true, numbering: none, [
+    $
+    sum_(k=1)^n k = (n(n+1)) / 2
+    $
+  ]
+)
+
+*Block with numbering:*
+
+As shown in @equation.
+
+$
+sum_(k=1)^n k = (n(n+1)) / 2
+$ <equation>
+
+*More information:*
+- #link("https://typst.app/docs/reference/math/equation/")
+
+
+= Citation
+
+You can use citations by using the `#cite` function with the key for the reference and adding a bibliography. Typst supports BibLateX and Hayagriva.
+
+```typst
+#bibliography("bibliography.bib")
+```
+
+Single citation #cite("Vaswani2017AttentionIA"). Multiple citation #cite("Vaswani2017AttentionIA", "hinton2015distilling"). In text #cite("Vaswani2017AttentionIA", brackets: false).
```

Comment:
This still uses the old string-based citation syntax. It would be good to update the package to be compatible with Typst 0.11.